test-apps/ninezone-test-app/package.json

```diff
@@ -36,10 +36,10 @@
     "classnames": "^2.3.1",
     "highlight.js": "~10.6.0",
     "raf-schd": "^4.0.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
-    "react-markdown": "^4.3.1",
-    "react-router-dom": "^5.2.0"
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-markdown": "^5.0.3",
+    "react-router-dom": "^5.1.7"
```

Comment:
did you mean to drop the version here?

Reply:
I did not. Probably I missed that when I was resolving merge conflicts.